### DISCUSSION TITLE
Dua device terhubung, 1 selalu gagal kirim pesan

### DISCUSSION THREAD

**Discussion by sanusi86** [kirimwa-id/forum#151] [Tanya Jawab] [answered]
Mas Rio,
Saya punya kendala. 2 Device (nomor wa berbeda) yg terhubung. Device pertama selalu sukses kirim pesan, tapi device kedua selalu gagal. Padahal menggunakan script yg sama untuk (send message).
Mohon solusinya.
Terima kasih

**Comment by rioastamal** [comment, marked as answer]
Tolong diperjelas errornya seperti apa. Kemudian status dari device kedua tersebut seperti apa query dengan `/v1/devices`. Coba logout dan discan ulang.

**Reply by sanusi86** [reply]
Tidak ada error, 
Tapi pesan selalu tidak terkirim.
Rescan sudah dicoba.

**Reply by rioastamal** [reply]
Kirimkan message_id untuk pesan yang gagal

**Reply by sanusi86** [reply]
Sekarang sudah normal kak dengan sendirinya. Saya tambah device baru sama pesan selalu gagal terkirim, setelah beberapa jam baru bisa. Mungkin proses sinkronnya membutuhkan waktu lama.
Terima kasih kak

**Reply by endrianur** [reply]
> Sekarang sudah normal kak dengan sendirinya. Saya tambah device baru sama pesan selalu gagal terkirim, setelah beberapa jam baru bisa. Mungkin proses sinkronnya membutuhkan waktu lama. Terima kasih kak

boleh liat full scriptnya yg sdh berhasil?
